src/main/java/com/mx/iwaconsolti/calculator/Services/CalculatorService.java

```diff
@@ -0,0 +1,94 @@
+package com.mx.iwaconsolti.calculator.Services;
+import com.mx.iwaconsolti.calculator.Interfaces.ArithmeticOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.util.Scanner;
+
+@Component
+public class CalculatorService {
+    private final ArithmeticOperation sumService;
+    private final ArithmeticOperation divService;
+
+    @Autowired
+    private SubsService subsService;
```

Comment:
If you are using interfaces for the other 2, you should make this also interfaces. You would need to use qualifier to indicate the bean name